patches/node/add_should_read_node_options_from_env_option_to_disable_node_options.patch

```diff
@@ -0,0 +1,68 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <samuel.r.attard@gmail.com>
+Date: Wed, 21 Jul 2021 13:40:59 -0700
+Subject: add should_read_node_options_from_env option to disable NODE_OPTIONS
+ parsing at runtime
```

Comment:
upstreamable?

Reply:
🤔  I don't _think_ so, the use case for a binary modifiable non-buildtime flag is very specific to Electron's usecase IMO

Reply:
i don't think the binary modifiable bit is inherent in this patch, it's just a flag to say whether or not to respect NODE_OPTIONS. seems like other node embedders might be interested in flagging it off, yeah?

Reply:
You can already flag it off at build time, the intersection between not doing custom builds and wanting to disable NODE_OPTIONS parsing at runtime is weirdly specific to our fuse implementation I think

Reply:
still seems not obviously useless to be able to disable at runtime. e.g. when embedding node as a shared library, & not wanting to rebuild.

Reply:
I'll talk to @codebytere 👍